.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: php
 
+sudo: false
+
 matrix:
   allow_failures:
     - php: hhvm
 
 php:
-  - 5.3
```

Comment:
This should not be removed.

AFAIK, php 5.3 is still supported by this library and this is not related to your PR.